RecoLocalCalo/EcalRecProducers/plugins/EcalCPUUncalibRecHitProducer.cc

```diff
@@ -79,42 +84,52 @@ void EcalCPUUncalibRecHitProducer::acquire(edm::Event const& event,
     cudaCheck(cudaMemcpyAsync(dest.data(), src, dest.size() * sizeof(type), cudaMemcpyDeviceToHost, ctx.stream()));
   };
 
+  if (!isPhase2_) {
+    auto const& eeRecHitsProduct = event.get(recHitsInEEToken_);
+    auto const& eeRecHits = ctx.get(eeRecHitsProduct);
+    recHitsEE_.resize(eeRecHits.size);
+  }
```

Comment:
Please move this part together with the other `if (!isPhase2_) { ... }` section.